docs/competition/teams/role-descriptions/reception.md

```diff
@@ -9,49 +9,63 @@ original:
 ---
 # Reception Volunteer
 
-Reception staff control entry to the building and validate people's competition
-tickets and/or wristbands. Each reception staffer has a desk, chair, and
-computer.
+Reception staff control entry to the building and validate people's Media Consent Forms (MCFs) and/or wristbands. Each reception staffer has their own desk and chair.
 
-There are four flavours of visitor to the competition: competitors,
-team-leaders, volunteers and true-visitors. **Everyone** must complete a media consent form (MCF) OR hold a valid ticket (which also counts as an MCF). Under 16's must have this form completed by a parent or guardian. If someone denies Media Consent, or is under age and doesn't have a form, please speak to the volunteer responsible for Managing Teams.
+**Everyone** must complete a MCF, Under 16's must have this form completed by a parent or guardian. If someone denies Media Consent, write down which team they are in, and hand them a red lanyard. If someone is under-16 and has no pre-signed form, speak to the Volunteer in charge.
 
 When someone enters the building, you should follow the procedure below.
 
-**Note: Training on how to use the scanning software will be given on the morning by the competition team coordinator.**
-
 ![Reception Desk Flow](../diagrams/reception-desk-flow.svg)
 
 At the start of the competition day, there will be four reception staffers
 covering the entrance. Once the initial influx of people is complete, this
-will reduce to two. Additionally, after the initial influx, receptionists
+will reduce. Additionally, after the initial influx, receptionists
 should allow people wearing SR wristbands to enter the building at will (they
 should check that people walking in have one, or follow the procedure above).
 
-**Note: All receptionists at the start of Saturday will be handling the arrivals desk too.**
-
-Throughout the competition reception staffers will encounter a variety of
+Throughout the competition, reception staffers will encounter a variety of
 members of the public enquiring what the event is, or otherwise receiving
-questions. Please answer as fully as possible, but refer any technical questions
-to helpdesk, or other unknown matters to other people in SR. Any member
-of the public should be allowed into the building if they sign a media consent
+questions. Please answer as fully as possible, but refer any technical or kit questions
+to the helpdesk. Any member of the public should be allowed into the building if they sign a media consent
 form.
 
-The 'crimper' role will also be performed by the receptionist. Please read this document as well for guidance on how to crimp correctly.
```

Comment:
We don't need crimps, but they still need to end up wearing a wristband, so something equating to this role is still valid